lib/pbio/drv/usb/stm32_usbd/usbd_desc.c

```diff
@@ -76,14 +87,143 @@ __ALIGN_BEGIN uint8_t USBD_DeviceDesc[USB_LEN_DEV_DESC] __ALIGN_END = {
     HIBYTE(USBD_VID),         /* idVendor */
     LOBYTE(USBD_PID),         /* idVendor */
     HIBYTE(USBD_PID),         /* idVendor */
-    0x00,                     /* bcdDevice rel. 2.00 */
-    0x02,
+    LOBYTE(USBD_VERSION_BCD), /* bcdDevice */
```

Comment:
Isn't this supposed to be the USB Specification Number which device complies too?